public/robots.txt

```diff
@@ -0,0 +1,13 @@
+# *
```

Comment:
we should use [robot api](https://nextjs.org/docs/app/api-reference/file-conventions/metadata/robots)

Reply:
We really don't need to, but that sounds a good alternative. Weird that if you search "robots.txt next" first result is https://nextjs.org/learn/seo/crawling-and-indexing/robots-txt which still has no reference to the new Api.

Good find!

Reply:
Personally I use it because it's an advantage to have everything in nextjs under the app dir. _after that it's a personal opinion_.